samples/04_gis_analysts_data_scientists/model_explainability_using_shap_for_tabular_data.ipynb

```diff
@@ -0,0 +1,534 @@
+{
```

Comment:
<p>Keep in mind the growing uset need --&gt; typo in word 'user'</p><p>Change sentence to "To support the growing need to make models more explainable, <code>arcgis.learn</code>  ..."</p><p>fully conneccted -&gt; typo in connected</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/977/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Done</p>